cartography/intel/gcp/storage.py

```diff
@@ -32,15 +32,13 @@ def get_gcp_buckets(storage: Resource, project_id: str, common_job_parameters) -
     :return: Storage response object
     """
     try:
+        buckets_list = []
         req = storage.buckets().list(project=project_id)
-        res = req.execute()
-        for item in res.get('items', []):
-            acl = item.get('acl', [])
```

Comment:
do we not use `acl` and `defaultObjectAcl` information anywhere?

Reply:
it is list[dict] object and by this logic overwrite all values and takes only last dict value from list. That’s why it is not useful

Reply:
ah! makes sense. merging for deployment.